src/commands/init.js

```diff
@@ -32,56 +32,41 @@ class InitCommand extends Command {
       },
     })
 
-    // const hasFlags = !isEmpty(flags)
-    let siteData
-    try {
-      siteData = await api.getSite({ siteId })
-    } catch (error) {
-      // silent api error
-      // TODO handle expected errors
-      // Throw unexpected ones
-    }
-
     // Add .netlify to .gitignore file
     await ensureNetlifyIgnore(site.root)
 
-    if (siteId && siteData && get(siteData, 'build_settings.repo_url') && !flags.force) {
```

Comment:
It's enough to check for `get(siteInfo, 'build_settings.repo_url')` since it will falsy if `siteId` and `siteData` are falsy